test/resources/SimpleAudioPlayer.js

```diff
@@ -0,0 +1,203 @@
+// This is only initialized when the Lambda is, so it is preserved across calls
+// It is NOT a real database, but can be used for testing, as JavaScript Lambdas tend to live for a few hours
+// Stay tuned for a more sophisticated example that uses DynamoDB
+var lastPlayedByUser = {};
```

Comment:
Is this class using var for any particular reason? , if not using const and let's for consistency make sense

Reply:
This is an exact copy of our simple audio player -  unfortunately, I use the old var style there, and as I would like to keep these in sync, changing this does not make sense.